targets/TARGET_RENESAS/TARGET_RZ_A1XX/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if ("RZA1H" IN_LIST MBED_TARGET_LABELS)
+    add_subdirectory(TARGET_RZ_A1H)
+elseif("RZA1UL" IN_LIST MBED_TARGET_LABELS)
+    add_subdirectory(TARGET_GR_LYCHEE)
+endif()
+
+target_include_directories(mbed-core
+        INTERFACE
+            .
+            common
+        )
```

Comment:
Fix indentation. Indentation should be 4 spaces and each level should be incremented by 4 spaces.
have a look at the indentation here: https://github.com/ARMmbed/mbed-os/pull/13968/files